eng/native/configureplatform.cmake

```diff
@@ -78,9 +78,9 @@ endif(CLR_CMAKE_HOST_OS STREQUAL Linux)
 if(CLR_CMAKE_HOST_OS STREQUAL Darwin)
     set(CLR_CMAKE_HOST_UNIX 1)
     set(CLR_CMAKE_HOST_OSX 1)
-    if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64)
+    if(CMAKE_OSX_ARCHITECTURES STREQUAL x86_64)
```

Comment:
Is it intentional for this to be plural? Can it have a list of more than one architecture?

Reply:
Yes. This is CMake's own variable and it can be set to both  x86_64 and arm64. See https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_ARCHITECTURES.html

We set it to only one of them in our build scripts.

Reply:
macOS binaries in Big-Sur seem to mostly be universal binaries supporting both architectures.  See also #41127.